internal/resources/metal/virtual_circuit/resource.go

```diff
@@ -37,6 +37,11 @@ func Resource() *schema.Resource {
 				Description: "UUID of Connection where the VC is scoped to",
 				ForceNew:    true,
 			},
+			"virtual_circuit_id": {
```

Comment:
This should also be `ForceNew: true`, because removing this attribute from config implies the user is changing from a shared connection VC to a dedicated connection VC.